pkg/api/controllers/dynamicschema/dynamicschema.go

```diff
@@ -84,7 +84,13 @@ func (c *Controller) add(dynamicSchema *v3.DynamicSchema) error {
 		schema.ID = dynamicSchema.Name
 	}
 	schema.Version = managementSchema.Version
-	c.Schemas.AddSchema(schema)
+	schema.DynamicSchemaVersion = dynamicSchema.ResourceVersion
```

Comment:
i dont understand this new dynamicSchemaVersion field. what is it? where is it set? what does it do?

Reply:
This is so that we don't double update a schema.  Inside Norman we compare this to the schema being updated, and if they match, we don't update.  See here: https://github.com/rancher/norman/pull/249/files#diff-7d1674a078b6d060648bb7b17fd9b4b9R134